jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/HttpReceiver.java

```diff
@@ -167,39 +128,40 @@ protected void receive()
      * This method takes case of notifying {@link org.eclipse.jetty.client.api.Response.BeginListener}s.
      *
      * @param exchange the HTTP exchange
-     * @return whether the processing should continue
      */
-    protected boolean responseBegin(HttpExchange exchange)
+    protected void responseBegin(HttpExchange exchange)
     {
-        if (!updateResponseState(ResponseState.IDLE, ResponseState.TRANSIENT))
-            return false;
+        if (LOG.isDebugEnabled())
+            LOG.debug("Response beginning on {}", this);
```

Comment:
```suggestion
        if (LOG.isDebugEnabled())
            LOG.debug("Invoking response begin for {} on {}", exchange, this);
```

I'd like to have logs to link the exchange and the receiver, so if possible always log both.
Here we are invoking because we are calling the `SerializedInvoker`, so the log should be "invoking ...".

Reply:
Done.